.github/workflows/pull-requests.yaml

```
@@ -37,5 +37,34 @@ jobs:
       - name: Build
         run: make build
 
+      - name: Upload installer artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: cozystack-installer
+          path: _out/assets/cozystack-installer.yaml
+
+  test:
```

actionlint (1.7.7)

job "test" needs job "build" which does not exist in this workflow

(job-needs)